kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/ErrorCertificationValidation.java

```diff
@@ -0,0 +1,61 @@
+package edu.arizona.kfs.fp.document.validation.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.document.validation.GenericValidation;
+import org.kuali.kfs.sys.document.validation.event.AttributedDocumentEvent;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.arizona.kfs.fp.businessobject.ErrorCertification;
+import edu.arizona.kfs.fp.document.GeneralErrorCorrectionDocument;
+import edu.arizona.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSKeyConstants;
+
+public class ErrorCertificationValidation extends GenericValidation {
+	
+	private static int NUM_ERROR_CERT_FIELDS = 4;
+	
+	protected static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ErrorCertificationValidation.class);
+	
+	public boolean validate(AttributedDocumentEvent event) {
+		GeneralErrorCorrectionDocument currDocument = (GeneralErrorCorrectionDocument)event.getDocument();
+		int numBlankFields = errorCertificationBlankFieldsCheck(currDocument);
+		LOG.debug("numBlankFields: " + numBlankFields);
+		
+		//if Error Certification tab is partially filled out
+		if((numBlankFields > 1) && (numBlankFields < KFSConstants.ErrorCertificationConstants.NUM_ERROR_CERT_FIELDS)) {
+			GlobalVariables.getMessageMap().putErrorForSectionId("document.errorCertification", KFSKeyConstants.ERROR_ERROR_CERT_FIELDS_REQ, "error.errorCert.fieldsReq");
+			return false;
+		}
+		
+		return true;
+	}
+
+	/**
+	 * This method goes through all the Error Certification fields looking for blank fields.
+	 * @param document
+	 * @return blankFieldCount the number of blank fields in the tab
+	 */
+	public int errorCertificationBlankFieldsCheck(GeneralErrorCorrectionDocument document) {
```

Comment:
Looking at this and super's same method, it appears this is duplicate code, with nothing special being done here. Any reason to not to delete this and let super handle it?